patches/chrome-browser-ui-views-tabs-tab_style_views.cc.patch

```diff
@@ -18,17 +18,7 @@ index 3e917d0207c85715805d77dcba8fecdffb53d1a9..3c3bbe59879d59f2923e9ad805482e4b
  
    SkPath path;
  
-@@ -608,8 +610,7 @@ TabStyle::SeparatorBounds GM2TabStyleViews::GetSeparatorBounds(
-       aligned_bounds.x() + corner_radius - separator_margin.right() -
-           separator_size.width(),
-       aligned_bounds.y() + (aligned_bounds.height() - separator_size.height() -
--                            separator_margin.bottom()) /
--                               2,
-+                            separator_margin.bottom()),
-       separator_size.width(), separator_size.height());
- 
-   separator_bounds.trailing = separator_bounds.leading;
```

Comment:
Are we sure about this? This patch was modified first here https://github.com/brave/brave-core/pull/19765/commits/7c2a68b4d0abb316c2d3d069aed0f2c749679c8a#diff-4e93b78a4670fc5580c2980084069fb8b61b39c420384bdb78c6cde573b7f234L21-L31 and now removed. Are these two changes congruent?

Reply:
@cdesouza-chromium Maybe you know about this one?

Reply:
This is a bit confounding, but I'm sure that at some point this patch had become implausible, but it appears that now it can be reverted entirely to what it was. It appears to be the case that there a fluid situation over this procedure. Thanks for noticing this, and apologies for the confusion. I'm reversing the unnecessary change, and fixing the patch.

Fixed in 046b8232a3228a5904febdd3d4be1858f65b285e.